appdata.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.geogebra.GeoGebra.desktop</id>
+  <name>GeoGebra</name>
+  <developer_name>GeoGebra</developer_name>
+  <summary>Dynamic mathematics software</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
In past they wanted Arch to drop the package that repackaged this tarball from repositories for some silly reasons. I'm not sure if we want to handle this here as well.